lms/djangoapps/mobile_api/testutils.py

```diff
@@ -92,9 +92,9 @@ def reverse_url(self, reverse_args=None, **kwargs):
             reverse_args.update({'username': kwargs.get('username', self.user.username)})
         return reverse(self.REVERSE_INFO['name'], kwargs=reverse_args)
 
-    def url_method(self, url, **kwargs):  # pylint: disable=unused-argument
+    def url_method(self, url, qargs={}, **kwargs):  # pylint: disable=unused-argument
         """Base implementation that returns response from the GET method of the URL."""
-        return self.client.get(url)
+        return self.client.get(url, qargs)
```

Comment:
Always use keyword arg style for arguments after the URL in HTTP methods: `self.client.get(url, qargs=qargs)`.  There are enough optional arguments to `self.client.get` that verifying that you're using the right one can be confusing if they're done positionally.